src/components/OAuthLogoutButton.tsx

```diff
@@ -59,47 +53,40 @@ export const OAuthLogoutButton: FC<OAuthLogoutButtonParams> = ({
   );
 };
 
-const defaultStyles = (theme: Theme) => {
-  const button: ViewStyle = {
+const defaultStyles = createStyles('OAuthLogoutButton', (theme: Theme) => ({
```

Comment:
We don't need to specify the type of Theme anymore in `(theme: Theme) => ...` because create styles types it for us so we can just do `(theme) => ...`